### DISCUSSION TITLE
Bar Chart - Label Position/Anchor based of Value

### DISCUSSION THREAD

**Discussion by socramm9** [chartjs/chartjs-plugin-datalabels#267] [Q&A] [answered]
Hey I am new working with ChartsJS and I am trying to get something like this to work: 
For example:

- if (value > 0) return "end" 
- if (value < 0) return "start"

This would result in something like this:
<img width="833" alt="Bildschirmfoto 2021-07-29 um 16 27 21" src="https://user-images.githubusercontent.com/52098754/127512023-327d5416-8139-4c8f-93fc-f171f931d710.png">

Is this Possible? I am currently only able to show the label on on side.

Thanks for a greet plugin :)

**Comment by socramm9** [comment]
Found the Answer right after I posted this haha 

<img width="337" alt="Bildschirmfoto 2021-07-29 um 16 52 34" src="https://user-images.githubusercontent.com/52098754/127514378-3e03255d-b85f-4d16-a2f1-9fe2b338442d.png">

**Comment by simonbrunel** [comment, marked as answer]
You can use [scriptable options](https://chartjs-plugin-datalabels.netlify.app/guide/options.html#scriptable-options):

```js
anchor: ({dataIndex, dataset}) => dataset.data[dataIndex] > 0 ? "end" : "start"
```
